os-images/AWS/photon/photon-3.json

```
@@ -0,0 +1,179 @@
+{
+  "_comment": "Build with `packer build photon.json`",
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "region": "{{ user `aws_region` }}",
+      "source_ami_filter": {
+        "filters": {
+          "architecture": "{{ user `ami_architecture` }}",
+          "virtualization-type": "hvm",
+          "name": "{{ user `ami_filter` }}",
+          "root-device-type": "ebs",
+          "state": "available"
+        },
+        "owners": [
+          "{{ user `ami_owner` }}"
+        ],
+        "most_recent": true
+      },
+      "subnet_filter": {
+        "filters": {
+          "tag:Name": "*-public-*"
+        },
+        "most_free": true,
+        "random": false
+      },
+      "vpc_filter": {
+        "filters": {
+          "tag:Name": "test"
+        }
+      },
+      "security_group_filter": {
+        "filters": {
+          "group-name": "kitchen-slave-auto-delete-test"
+        }
+      },
+      "instance_type": "{{ user `instance_type` }}",
+      "ssh_interface": "private_ip",
+      "ssh_username": "{{ user `ssh_username` }}",
+      "ssh_keypair_name": "kitchen",
+      "ssh_private_key_file": "~/.ssh/kitchen.pem",
+      "ami_name": "{{ user `ami_name_prefix` }}/{{ user `build_type` }}/{{ user `ami_name_suffix` }}/{{ isotime \"2006-01-02-15-04-05\" }}",
+      "ami_groups": [
+        "all"
+      ],
+      "ami_description": "{{ user `build_type` | upper }} Image of {{ user `os_name` }} {{ user `os_version` }} {{ user `os_arch` }}",
+      "tags": {
+        "Name": "{{ user `build_type` | upper }} // {{ user `os_name` }} {{ user `os_version` }} {{ user `os_arch` }}",
+        "OS-Name": "{{ user `os_name` }}",
+        "OS-Version": "{{ user `os_version` }}",
+        "OS-Arch": "{{ user `os_arch` }}",
+        "Build-Date": "{{ isotime }}",
+        "Build-Type": "{{ user `build_type` | upper }}",
+        "created-by": "packer",
+        "Owner": "SRE",
+        "Provision-State-Name": "{{ user `state_name` }}",
+        "Promoted": false
+      },
+      "run_tags": {
+        "Name": "Packer {{ user `build_type` | upper }} {{ user `os_name` }} {{ user `os_version` }} {{ user `os_arch` }} Builder",
+        "created-by": "packer",
+        "Owner": "SRE"
+      },
+      "launch_block_device_mappings": [
+        {
+          "device_name": "/dev/xvda",
+          "volume_size": 40,
+          "volume_type": "gp3",
+          "delete_on_termination": true
+        }
+      ]
+    }
+  ],
+  "provisioners": [
+    {
+      "type": "shell",
+      "inline_shebang": "/bin/bash -ex",
+      "inline": [
+        "find /etc/yum.repos.d -type f -exec sed -i 's!dl.bintray.com/vmware!packages.vmware.com/photon/$releasever!' {} ';'",
```

```suggestion
        "growpart /dev/nvme0n1 3",
        "resize2fs /dev/nvme0n1p3",
        "df -h|grep /$|grep '39G\\|40G'",
        "systemctl mask tmp.mount",
        "find /etc/yum.repos.d -type f -exec sed -i 's!dl.bintray.com/vmware!packages.vmware.com/photon/$releasever!' {} ';'",
```

Adding this suggestion, based off of modifications we had to do to Photon OS 3 via https://github.com/saltstack/salt-jenkins/pull/1725 (I'm not sure how different Photon OS 4 functionality is)